storage.go

```diff
@@ -547,7 +547,7 @@ func (storage DBStorage) DatabaseCleanup() error {
 }
 
 func dropTableStatement(tableName string) string {
-	return "DROP TABLE " + tableName + ";"
+	return "DROP TABLE " + tableName + " CASCADE;"
```

Comment:
why is it needed there?

Reply:
I'm actually thinking of opening an issue for refactoring this code because it only drops the initial tables, not the ones created through migrations. WDYT?